assets/src/edit-story/elements/text/output.js

```diff
@@ -46,8 +46,8 @@ function TextOutput({
     fontFamily: generateFontFamily(fontFamily, fontFallback),
     fontWeight: fontWeight ? fontWeight : null,
     background: backgroundColor,
-    color,
-    lineHeight,
+    color:color,
+    lineHeight:lineHeight,
```

Comment:
Weird, I thought this automatic change from shorthand and back was gone 🤔 
Need to look into the linter rules again.